packages/themes/default/src/index.ts

```diff
@@ -70,11 +70,11 @@ export const DEFAULT = KoliBri.createTheme('default', {
 		kol-tooltip-wc .tooltip-content {
 			border-radius: var(--border-radius);
 			line-height: 1.5;
-			padding: var(--spacing) calc(var(--spacing) * 1.5);
+			padding: 0.5rem 0.75rem;
```

Comment:
Ich überlege, ob hier nicht `em` besser wäre. Wir haben immer lokal in den Komponenten `em` verwenden, damit diese via Font-Size proportional skalierbar sind.

Reply:
Ich persönlich habe immer `rem` präferiert, weil es auch skalierbar ist, aber über die font-size vom Dokument. Eine angepasst Schriftgröße (vom Browser, Web Extensions, etc.) wird also berücksichtigt.

`em` empfand ich immer etwas "unvorhersehbar" bzw. es kann leicht zu ungewollten Side-Effects kommen.
Sollen Tooltips z.B. wirklich verschiedene Innenabstände haben, je nachdem wo auf der Seite sie verwendet werden?

Das Argument der Vereinheitlichung kann ich jedoch gut nachvollziehen. Soll ich alle `rem`s im Default Theme zu `em`s konvertieren?

Reply:
Passt: margin, padding und gap `rem` Design System Aspekt